src/test/java/org/mule/test/extension/file/common/AbstractPostActionGroupTestCase.java

```diff
@@ -148,6 +148,7 @@ private class ConcretePostActionGroup extends AbstractPostActionGroup {
     private String moveToDirectory;
     private String renameTo;
     private boolean isAutoDelete;
+    private boolean overwrite = false;
```

Comment:
Shouldn’t you have to test the true value also? It may be better if this value is in a constructor's parameter, what do you think?

Reply:
Oh, this is a mistake. I merged the latest branch and something messed up. 
This one should be removed. I will update it.